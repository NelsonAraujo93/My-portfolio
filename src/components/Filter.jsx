import PropTypes from 'prop-types';
import { BallTriangle } from 'react-loader-spinner';
import { useDispatch, useSelector } from 'react-redux';
import { NavLink, Outlet, useNavigate } from 'react-router-dom';
import '../styles/filter.css';
import {
  AiFillCloseCircle, AiOutlineArrowLeft, AiOutlineHome, AiOutlineMenu,
} from 'react-icons/ai';
import { useEffect, useState } from 'react';
import { getAllTeamsAPI, selectTeam } from '../redux/teams/teamsSlice';
import { getAllGamesAPI } from '../redux/games/gamesSlice';
import SideMenuItem from './SideMenuItem';

const SideMenu = ({ teams, closeMenu, chooseTeam }) => (
  <ul data-testid="side-container" className="side-teams">
    {
      teams.map((team) => (
        <SideMenuItem closeMenu={closeMenu} chooseTeam={chooseTeam} team={team} key={`${team.id}-key`} />
      ))
    }
  </ul>
);

SideMenu.propTypes = {
  teams: PropTypes.instanceOf(Array).isRequired,
  closeMenu: PropTypes.func.isRequired,
  chooseTeam: PropTypes.func.isRequired,
};

function Filter() {
  const [home, setHome] = useState(true);
  const teamsStatus = useSelector((store) => store.teams.status);
  const gamesStatus = useSelector((store) => store.games.status);
  const [openMenu, setOpenMenu] = useState(false);
  const dispatch = useDispatch();
  const navigate = useNavigate();
  const { teams } = useSelector((store) => store.teams);

  useEffect(() => {
    if (teamsStatus === 'failed') return;
    if (gamesStatus === 'failed') return;
    if (teams) return;
    dispatch(getAllTeamsAPI());
    dispatch(getAllGamesAPI());
  }, [selectTeam]);
  return (
    <>
      {
        gamesStatus && teamsStatus === 'loading' && (
          <div className="loading-screen">
            <BallTriangle
              height={150}
              width={150}
              radius={5}
              color="#712041"
              ariaLabel="ball-triangle-loading"
              visible
            />
          </div>
        )
      }
      <section
        className={openMenu ? 'side-menu active' : 'side-menu'}
      >
        <div className="menu-btn-container">
          <button type="button" className="btn-menu" onClick={() => setOpenMenu(false)}>
            <AiFillCloseCircle size={26} color="#fff" />
          </button>
        </div>
        {
          teams && (
            <SideMenu
              teams={teams}
              closeMenu={() => setOpenMenu(false)}
              chooseTeam={() => { setHome(false); }}
            />
          )
        }
      </section>
      <section className="navbar">
        {
          !home ? (
            <div className="menu-btn-container">
              <button type="button" className="btn-menu">
                <AiOutlineArrowLeft
                  size={26}
                  color="#fff"
                  onClick={() => {
                    setHome(true);
                    dispatch(selectTeam(null));
                    navigate('/');
                  }}
                />
              </button>
            </div>
          ) : (
            <div className="menu-btn-container">
              <button type="button" className="btn-menu" onClick={() => setOpenMenu(true)}>
                <AiOutlineMenu size={26} color="#fff" />
              </button>
            </div>
          )
        }
        <h4>NBA SEASON 2022 - 2023</h4>
        <div className="menu-btn-container">
          <NavLink
            to="/"
          >
            <button type="button" className="btn-menu">
              <AiOutlineHome size={26} color="#fff" />
            </button>
          </NavLink>
        </div>
      </section>
      {
        gamesStatus && teamsStatus === 'succeed' && (
          <Outlet />
        )
      }
    </>
  );
}

export default Filter;
