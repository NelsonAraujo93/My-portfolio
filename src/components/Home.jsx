import '../styles/home.css';

export default function Home() {
  return (
    <section>
      <div className="wrapper">
        <h1>Wellcome to world metrics</h1>
        <p>Choose one of the options or use the filter bellow</p>
      </div>
    </section>
  );
}
