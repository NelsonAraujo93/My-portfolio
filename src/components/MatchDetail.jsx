import { useEffect } from 'react';
import { BallTriangle } from 'react-loader-spinner';
import { useDispatch, useSelector } from 'react-redux';
import { useParams } from 'react-router-dom';
import { selectGameMatchById } from '../redux/games/gamesSlice';
import '../styles/team.css';
import '../styles/match.css';

function MatchDetail() {
  const { selectedGame, fullGames } = useSelector((state) => state.games);
  const { id } = useParams();
  const dispatch = useDispatch();
  const options = {
    weekday: 'long', year: 'numeric', month: 'long', day: 'numeric',
  };

  useEffect(() => {
    if (fullGames) {
      dispatch(selectGameMatchById(id));
    }
  }, [dispatch, fullGames]);

  if (!fullGames || !selectedGame) {
    return (
      <div className="loading-screen">
        <BallTriangle
          height={150}
          width={150}
          radius={5}
          color="#712041"
          ariaLabel="ball-triangle-loading"
          visible
        />
      </div>
    );
  }

  const date = new Date(selectedGame.date.start);

  return (
    <>
      <section className="team-header match-details">
        <div className="home">
          <h3 className="tittle">{selectedGame.teams.home.code}</h3>
          <img src={selectedGame.teams.home.logo} alt={`${selectedGame.teams.home.code}-logo`} />
          <h2>{selectedGame.scores.home.points}</h2>
        </div>
        <div className="visitor">
          <h3 className="tittle">{selectedGame.teams.visitors.code}</h3>
          <img src={selectedGame.teams.visitors.logo} alt={`${selectedGame.teams.visitors.code}-logo`} />
          <h2>{selectedGame.scores.visitors.points}</h2>
        </div>
      </section>
      <section className="match-data">
        <h1 className="subtitle">Details</h1>
        <h2>{date.toLocaleDateString('en-US', options)}</h2>
      </section>
    </>
  );
}

export default MatchDetail;
