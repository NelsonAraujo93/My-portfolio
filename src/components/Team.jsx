import { useEffect } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { useParams } from 'react-router-dom';
import { getTeamGamesAPI, getTeamPlayerAPI, selectTeamById } from '../redux/teams/teamsSlice';
import '../styles/team.css';
import Player from './Player';
import Match from './Match';

function Team() {
  const {
    selectedTeam, selectedTeamGames, selectedTeamPlayers, teams,
  } = useSelector((store) => store.teams);
  const { id } = useParams();
  const dispatch = useDispatch();
  let color = false;

  useEffect(() => {
    if (teams && !selectedTeam) {
      dispatch(selectTeamById(id));
      dispatch(getTeamGamesAPI(id));
      dispatch(getTeamPlayerAPI(id));
    }
  }, [teams, selectedTeam, selectedTeamGames, selectedTeamPlayers]);

  if (!selectedTeam || !selectedTeamGames || !selectedTeamPlayers) {
    return (
      <h1>Select a team from side menu</h1>
    );
  }

  const createMatch = (item, col) => {
    color = col ? !color : color;
    return (
      <Match
        key={item.id}
        data={item}
        team={false}
        odd={color}
      />
    );
  };

  return (
    <>
      <section className="team-header">
        <h3 className="tittle">{selectedTeam.name}</h3>
        <img src={selectedTeam.logo} alt={`${selectedTeam.name}-logo`} />
      </section>
      <section className="team-players">
        <h1 className="subtitle">Players</h1>
        <div className="players-list">
          {
            selectedTeamPlayers.map((item, i) => (
              <Player
                key={item.id}
                data={item}
                odd={i % 3 === 0}
              />
            ))
          }
        </div>
      </section>
      <section className="team-matches">
        <h1 className="subtitle">Last Matches</h1>
        <div className="matches-list">
          {
            selectedTeamGames.map((item, i) => (
              createMatch(item, (i + 1) % 2 === 0, i)
            ))
          }
        </div>
      </section>
    </>
  );
}

export default Team;
