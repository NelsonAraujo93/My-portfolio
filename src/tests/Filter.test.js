import { configureStore, createSlice } from '@reduxjs/toolkit';
import { render, screen } from '@testing-library/react';
import { Provider } from 'react-redux';
import { createBrowserRouter, RouterProvider } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ErrorPage from '../components/ErrorPage';
import Filter from '../components/Filter';
import Home from '../components/Home';
import MatchDetail from '../components/MatchDetail';
import Team from '../components/Team';

describe('Tests for <Filter /> component', () => {
  const router = createBrowserRouter([
    {
      path: '/',
      element: <Filter />,
      errorElement: <ErrorPage />,
      children: [
        {
          index: true,
          element: <Home />,
        },
        {
          path: '/team/:id',
          element: <Team />,
        },
        {
          path: '/match/:id',
          element: <MatchDetail />,
        },
      ],
    },
  ]);
  const initialState = {
    teams: [
      {
        id: '1',
        name: 'Hawks',
        code: 'ATL',
        logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
        nbaFranchise: true,
      }, {
        id: '2',
        name: 'Hawks',
        code: 'ATL',
        logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
        nbaFranchise: true,
      },
    ],
    status: 'succeed',
    error: '',
  };
  const teamsSlice = createSlice({
    name: 'teams',
    initialState,
    reducers: {
      changeStatus: (state, { payload }) => ({
        ...state, status: payload,
      }),
    },
  });
  const initialState2 = {
    games: [{
      id: '1',
      name: 'Hawks',
      code: 'ATL',
      status: { long: 'Finished' },
      teams: {
        home: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
        visitors: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
      },
      scores: {
        visitors: {
          points: 123,
        },
        home: {
          points: 123,
        },
      },
    }, {
      id: '2',
      name: 'Hawks',
      code: 'ATL',
      status: { long: 'Finished' },
      teams: {
        home: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
        visitors: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
      },
      scores: {
        visitors: {
          points: 123,
        },
        home: {
          points: 123,
        },
      },
    },
    ],
    fullGames: [{
      id: '1',
      name: 'Hawks',
      code: 'ATL',
      status: { long: 'Finished' },
      teams: {
        home: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
        visitors: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
      },
      scores: {
        visitors: {
          points: 123,
        },
        home: {
          points: 123,
        },
      },
    }, {
      id: '2',
      name: 'Hawks',
      code: 'ATL',
      status: { long: 'Finished' },
      teams: {
        home: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
        visitors: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
      },
      scores: {
        visitors: {
          points: 123,
        },
        home: {
          points: 123,
        },
      },
    },
    ],
    inCommingGames: [{
      id: '1',
      name: 'Hawks',
      code: 'ATL',
      status: { long: 'Finished' },
      teams: {
        home: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
        visitors: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
      },
      scores: {
        visitors: {
          points: 123,
        },
        home: {
          points: 123,
        },
      },
    }, {
      id: '2',
      name: 'Hawks',
      code: 'ATL',
      status: { long: 'Finished' },
      teams: {
        home: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
        visitors: {
          logo: 'https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png',
          nickname: 'ATL',
          code: 'ATL',
        },
      },
      scores: {
        visitors: {
          points: 123,
        },
        home: {
          points: 123,
        },
      },
    },
    ],
    status: 'succeed',
    error: '',
  };
  const gamesSlice = createSlice({
    name: 'games',
    initialState: initialState2,
    reducers: {
      changeStatusGames: (state, { payload }) => ({
        ...state, status: payload,
      }),
    },
  });
  const store = configureStore({
    reducer: {
      teams: teamsSlice.reducer,
      games: gamesSlice.reducer,
    },
  });
  const wrapper = () => (
    <Provider store={store}>
      <RouterProvider router={router} />
    </Provider>
  );
  it('Should match the snapshot', () => {
    expect(renderer.create(wrapper(<Filter />)).toJSON()).toMatchSnapshot();
  });
  it('Should render two items', () => {
    render(wrapper(<Filter />));
    expect(screen.queryByTestId('side-container').children.length).toBe(2);
  });
});
