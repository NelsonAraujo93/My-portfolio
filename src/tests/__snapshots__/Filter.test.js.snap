// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <Filter /> component Should match the snapshot 1`] = `
Array [
  <section
    className="side-menu"
  >
    <div
      className="menu-btn-container"
    >
      <button
        className="btn-menu"
        onClick={[Function]}
        type="button"
      >
        <svg
          color="#fff"
          fill="currentColor"
          height={26}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "#fff",
            }
          }
          viewBox="0 0 1024 1024"
          width={26}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 0 1-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
          />
        </svg>
      </button>
    </div>
    <ul
      className="side-teams"
      data-testid="side-container"
    >
      <li
        className="side-menu-item"
        style={
          Object {
            "background": "linear-gradient(90deg,  #1e1e1e 1%, #1e1e1e, 90%, #1e1e1e 99%)",
          }
        }
      >
        <button
          onClick={[Function]}
          type="button"
        >
          <a
            className="side-link"
            href="/team/1"
            onClick={[Function]}
          >
            <h3>
              ATL
            </h3>
            <p>
              Hawks
            </p>
            <div
              className="team-logo"
            >
              <div
                className="hidden-text"
              >
                <h3>
                  ATL
                </h3>
                <p>
                  Hawks
                </p>
              </div>
              <img
                alt="Hawks-logo"
                src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
              />
            </div>
          </a>
        </button>
      </li>
      <li
        className="side-menu-item"
        style={
          Object {
            "background": "linear-gradient(90deg,  #1e1e1e 1%, #1e1e1e, 90%, #1e1e1e 99%)",
          }
        }
      >
        <button
          onClick={[Function]}
          type="button"
        >
          <a
            className="side-link"
            href="/team/2"
            onClick={[Function]}
          >
            <h3>
              ATL
            </h3>
            <p>
              Hawks
            </p>
            <div
              className="team-logo"
            >
              <div
                className="hidden-text"
              >
                <h3>
                  ATL
                </h3>
                <p>
                  Hawks
                </p>
              </div>
              <img
                alt="Hawks-logo"
                src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
              />
            </div>
          </a>
        </button>
      </li>
    </ul>
  </section>,
  <section
    className="navbar"
  >
    <div
      className="menu-btn-container"
    >
      <button
        className="btn-menu"
        onClick={[Function]}
        type="button"
      >
        <svg
          color="#fff"
          fill="currentColor"
          height={26}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "#fff",
            }
          }
          viewBox="0 0 1024 1024"
          width={26}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
          />
        </svg>
      </button>
    </div>
    <h4>
      NBA SEASON 2022 - 2023
    </h4>
    <div
      className="menu-btn-container"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <button
          className="btn-menu"
          type="button"
        >
          <svg
            color="#fff"
            fill="currentColor"
            height={26}
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "#fff",
              }
            }
            viewBox="0 0 1024 1024"
            width={26}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M946.5 505L560.1 118.8l-25.9-25.9a31.5 31.5 0 0 0-44.4 0L77.5 505a63.9 63.9 0 0 0-18.8 46c.4 35.2 29.7 63.3 64.9 63.3h42.5V940h691.8V614.3h43.4c17.1 0 33.2-6.7 45.3-18.8a63.6 63.6 0 0 0 18.7-45.3c0-17-6.7-33.1-18.8-45.2zM568 868H456V664h112v204zm217.9-325.7V868H632V640c0-22.1-17.9-40-40-40H432c-22.1 0-40 17.9-40 40v228H238.1V542.3h-96l370-369.7 23.1 23.1L882 542.3h-96.1z"
            />
          </svg>
        </button>
      </a>
    </div>
  </section>,
  <section>
    <h1
      className="subtitle"
    >
      Last matches
    </h1>
    <div
      className="matches-list"
    >
      <button
        className="match-card"
        onClick={[Function]}
        type="button"
      >
        <h2
          className="match-status"
        >
          Finished
        </h2>
        <div
          className="vs"
        >
          <div
            className="home"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
          <div
            className="visitor"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
        </div>
      </button>
      <button
        className="match-card pair"
        onClick={[Function]}
        type="button"
      >
        <h2
          className="match-status"
        >
          Finished
        </h2>
        <div
          className="vs"
        >
          <div
            className="home"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
          <div
            className="visitor"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
        </div>
      </button>
    </div>
  </section>,
  <section>
    <h1
      className="subtitle"
    >
      Comming matches
    </h1>
    <div
      className="matches-list"
    >
      <button
        className="match-card pair"
        onClick={[Function]}
        type="button"
      >
        <h2
          className="match-status"
        >
          Finished
        </h2>
        <div
          className="vs"
        >
          <div
            className="home"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
          <div
            className="visitor"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
        </div>
      </button>
      <button
        className="match-card"
        onClick={[Function]}
        type="button"
      >
        <h2
          className="match-status"
        >
          Finished
        </h2>
        <div
          className="vs"
        >
          <div
            className="home"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
          <div
            className="visitor"
          >
            <img
              alt="ATL-logo"
              className="match-icon"
              src="https://upload.wikimedia.org/wikipedia/fr/thumb/6/…on_logo.svg/1024px-Celtics_de_Boston_logo.svg.png"
            />
            <h2>
              123
            </h2>
            <h3>
              ATL
            </h3>
          </div>
        </div>
      </button>
    </div>
  </section>,
]
`;
